package.json

```diff
@@ -78,16 +78,13 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
-    "fbjs": "^0.8.16",
     "fela": "^6.1.7",
     "fela-plugin-fallback-value": "^5.0.17",
     "fela-plugin-placeholder-prefixer": "^5.0.18",
     "fela-plugin-prefixer": "^5.0.18",
     "fela-plugin-rtl": "^1.0.6",
-    "hoist-non-react-statics": "^2.5.0",
     "keyboard-key": "^1.0.1",
     "lodash": "^4.17.10",
-    "normalize.css": "^8.0.0",
```

Comment:
Only used to create teams theme, which includes a stringified version of this module.